### DISCUSSION TITLE
DietPi - docker images

### DISCUSSION THREAD

**Discussion by isarrider** [MichaIng/DietPi#5924] [Image Request]
Hi,

quick question:
In
https://dietpi.com/downloads/images/
there are
DietPi_Container-ARMv8-*
images....
Is this sth focused on users only using containers?
I could not really find anything regarding...

BR,
Alex

**Comment by Joulinar** [comment]
These containers we use to build our images. They are not designed to run on docker. Actually we don't have Docker images.

**Comment by isarrider** [comment]
ah, thanks...

**Reply by Joulinar** [reply]
We thought about it but there was no real use case for a Docker container with DietPi.

**Reply by isarrider** [reply]
ah, I thought ab it the other way round...
Use basically only the bare minimum and have for everything else a container runtime (like docker or podman),
but as dietPi is so leightway anyway prob no need...

**Comment by MichaIng** [comment]
> These containers we use to build our images.

Used to build software packages, images are built directly on Ubuntu GitHub runners 😉. In theory the container images can be used as basis for Docker images as well, but they contain an fstab, hence require direct assess to the underlying filesystem/partition devices and have no network stack installed, hence rely on host network at first. So the Docker container would need to be setup in a way this works, or the image adjusted.

However, those images contain systemd, APT and other basic Debian system infrastructure which isn't required on common container use cases, and are hence not really lightweight, compared to e.g. Alpine Linux based containers. And, speaking of lightweight: Of course running any container, you need a host OS which drives a container engine to run the container with, no nothing is more lightweight than running software on the host OS directly, which is what DietPi aims to make easy.

In a first step, I will remove the fstab from our container images, which make them better suitable for common container use cases.
